docs/releases.md

```diff
@@ -1,5 +1,97 @@
 # DietPi Releases
 
+## December 2021 (Beta Version 7.9) {: #december-2021-version-79 }
+
+### Overview
+
+**DietPi Beta v7.9 is ready for testing ! While we are in beta, we are working at full speed to roll out DietPi improvments and new features.**
+
+Take us for a spin, and let us know if you hit any road bumps along the way. The fun is in the journey.
+
+**Ready to get started ?** Find here the steps how to use the DietPi Beta branch [branch system](https://github.com/MichaIng/DietPi/blob/master/BRANCH_SYSTEM.md).
+
+### Announcement
+
+!!! Warning "**Debian "stretch" support**"
+
+    Debian 9.13 "stretch" was released July 18th, 2020, and it has been initially superseded by Debian 10 (buster), and then Debian 11 (bullseye).
```

Comment:
`9.13` => `9`, some people may be on older point releases, but all of 9 is Stretch.